changelogs/fragments/871.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - zabbix_proxy_dbpassword - when zabbix_proxy_dbpassword_hash_method is set to anything other than md5 then do not hash the password with md5 so you could use postgresql scram-sha-256 hashing method
+  - zabbix_proxy_dbpassword_hash_method - add variable to control whether you want postgresql user password to be hashed with md5 or want to use db default
```

```suggestion
  - zabbix_proxy role - Add variable zabbix_proxy_dbpassword_hash_method to control whether you want postgresql user password to be hashed with md5 or want to use db default. When zabbix_proxy_dbpassword_hash_method is set to anything other than md5 then do not hash the password with md5 so you could use postgresql scram-sha-256 hashing method. 
```
I'd suggest following re-wording.